tests/unit/cn/kblock_builder_test.c

```diff
@@ -192,6 +193,18 @@ test_setup(struct mtf_test_info *lcl_ti)
     return 0;
 }
 
+int
+test_setup2(struct mtf_test_info *lcl_ti)
```

Comment:
Better function name. Function names are no longer allowed to end in 2 :).

Reply:
Haha.  I have a patch somewhere that renames kvset_create2 :)